src/Atoms/Switch/Switch.test.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react'
+import { render, fireEvent } from '@testing-library/react'
+import '@testing-library/jest-dom/extend-expect'
+import Switch from './index'
+
+const StateHolder = ({ children = () => {} }) => {
+  const [checked, setChecked] = useState(false)
+
+  return <>
+    {checked ? 'Yup' : 'Nope'}
```

Comment:
I guess that works, but why not going for `data-testid` + `getByTestId` instead ?